src.mk

```diff
@@ -651,6 +651,7 @@ JNI_NATIVE_SOURCES =                                          \
   java/rocksjni/checkpoint.cc                                 \
   java/rocksjni/clock_cache.cc                                \
   java/rocksjni/cache.cc                                      \
+  java/rocksjni/columncamilydescriptor.cc                     \
```

Comment:
`columncamilydescriptor.cc` -> `columnfamilydescriptor.cc`

Reply:
Just wondering if this indicates that tests have not been run yet with the Make build?

Reply:
I think I name the file wrongly and then I copied error everywhere. 

Test are running : https://github.com/facebook/rocksdb/actions/runs/8565557203/job/23473783606?pr=12505